config_files/config_de_novo_irma.yaml

```diff
@@ -44,8 +44,14 @@ Tools:
   bin: /sw/apps/bioinfo/trinity/2014-07-17/irma/ 
   options: []
  BUSCO:
-  bin: /sw/apps/bioinfo/BUSCO/1.1b1/irma/BUSCO
+  bin: /sw/apps/bioinfo/BUSCO/2.0.1/irma/BUSCO
   options:  ["-m", "genome", "-f"]
  kmergenie:
   bin: /lupus/ngi/production/latest/KmerGenie/kmergenie
   options: [""]
+ FRC:
+  bin: /lupus/ngi/staging/latest/resources/nougat_singularity/FRC
+  options: ""
+ qaTools:
+  bin: /lupus/ngi/staging/latest/resources/nougat_singularity/qaCompute 
+  options: ""
```

Comment:
@sylvinite Is this the correct path?

Reply:
Use /lupus/ngi/staging/latest/resources/singularity/nougat/qaCompute instead. Makes less redundancy for methylseq pipeline and other projects

Reply:
Should it be `staging` ? It will work, but wouldn't `production` be more appropriate ?

Reply:
Good points @sylvinite and @senthil10. How about: ```/lupus/ngi/production/latest/resources/singularity/nougat/qaCompute```

Reply:
Looks good to me 👍

Reply:
:+1: